docs/user-guide/quick-start/contributor-api.md

```diff
@@ -265,19 +309,24 @@ to study and link data to samples.
     2. Click on the endpoint  to **Create a link between a group of expression objects and a group of 
    samples objects**: 
    `/api/v1/as-curator/integration/link/expression/group/{sourceId}/to/sample/group/{targetId}`
-   ![Data Integration](quick-start-images/post-data-integration.png)
+  
+    ![Data Integration](quick-start-images/post-data-integration.png)
+    <figcaption>Click on Expression integration as Curator to find the endpoints to link experimental data with samples (and the study). Select the endpoint ==/api/v1/as-curator/integration/link/expression/group/{sourceId}/to/sample/group/{targetId}==</figcaption>
 
 3. **Enter Accession Details:** Add the relevant information, including the data accession number and sample group accession
 number, and click **Execute.**
    ![Accession](quick-start-images/post-data-accession.png)
+   <figcaption>Add the details of the accession numbers corresponding to the experimental data (<strong>GSF1147049</strong>) and the sample group object (<strong>GSF1147034</strong>). By default, the linking attribute will be the column <strong>Sample source ID</strong>, but you can customize it </figcaption>
```

Comment:
Could we please use certain attribute name? Sample Source ID instead of Sample source ID